### PR TITLE
Updated all dependencies.

### PR DESCRIPTION
**Pull request by bhamodi**
Updated all the dependencies. Also, not sure why we were previously pointing to version 200.4.0 of npm-shrinkwrap. The latest release on npm is 5.4.0.

### CODE REVIEW

**Comment by kkirsche** [comment]
This reverts the node version from 0.12.4 to 0.10.29 in your shrinkwrap. Also, if there are no benefits to the updates (such as features or fixes needed) they generally aren't updated

File: test-infra/npm-shrinkwrap.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "bootstrap",
   "version": "3.3.4",
-  "npm-shrinkwrap-version": "200.4.0",
-  "node-version": "v0.12.4",
+  "npm-shrinkwrap-version": "5.4.0",
+  "node-version": "v0.10.29",
```
Comment (kkirsche) [inline test-infra/npm-shrinkwrap.json]: This reverted from v0.12.4

**Comment by kkirsche** [comment]
Also the choice of shrinkwrap 200 is from twbs/bootstrap#15355 to fix a bug

**Comment by clarkdemvher** [comment]
On Jun 15, 2015 4:28 AM, "Baraa Hamodi" <notifications@github.com> wrote:
>
> Updated all the dependencies. Also, not sure why we were previously
pointing to version 200.4.0 of npm-shrinkwrap. The latest release on npm is
5.4.0.

**Comment by bhamodi** [comment]
Thanks for the clarification @kkirsche! Updated the PR.

**Comment by bhamodi** [comment]
Hey Bootstrap team! I was just wondering when/if this PR will be merged in soon? :smiley:

**Comment by kkirsche** [comment]
Hey @bhamodi, focus is currently on prepping for V4 in August but a team member will merge when they have the time to review the PR

**Comment by cvrebert** [comment]
@bhamodi Thanks! Sorry for the delay.

**Comment by bhamodi** [comment]
Thanks @cvrebert!

**Pull request closed without merging**